crates/node/src/catchup.rs

```diff
@@ -391,6 +391,7 @@ impl Node {
                         .install_application_from_url(
                             change.source.to_string().parse()?,
                             change.version,
+                            Vec::new()
```

Comment:
we should receive application metadata from the peer, you can introduce a comment here to expand the type of `change` to include metadata information